vulcanai2/models/basenetwork.py

```diff
@@ -276,11 +332,11 @@ def fit(self, train_loader, val_loader, epochs, retain_graph=False, valid_interv
 
         try:
             for epoch in trange(self.epoch, epochs, desc='Epoch: ', ncols=80):
-                train_loss, train_acc = self.train_epoch()
+                train_loss, train_acc = self._train_epoch()
```

Comment:
I think both `train_epoch` and `validate` should have consistent names either `_train_epoch` and `_validate` or without post underscores

Reply:
don't I have that? I think that was a mistake while merging conflicts. will fix.